### PR TITLE
[FAB-18492] Fix broken links in the 2.0 release docs

### PR DESCRIPTION
**Pull request by nikhil550**
Signed-off-by: Nikhil Gupta <ngupta@symbridge.com>

#### Type of change

- Bug fix
- Documentation update

#### Description

Fix broken links found by link report in 2.0 docs

### CODE REVIEW

File: docs/source/developapps/transactionhandler.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ shows you the exact form of these handlers.
 
 Once a handler has been added to the smart contract, it will be invoked during
 transaction processing. During processing, the handler receives `ctx`, the
-[transaction context](./transationcontext.md), performs some processing, and
+[transaction context](./transationcontext.html), performs some processing, and
```
Comment (denali49) [inline docs/source/developapps/transactionhandler.md]: fix typo of './transationcontext.html as 'transaction has a typo breaking the link.

**Review by denali49 — CHANGES_REQUESTED** [review]
see inline comment on the single typo, the rest are checked and good.  Thanks

File: docs/source/developapps/transactionhandler.md (hunk 1 above)
Comment (nikhil550) [reply docs/source/developapps/transactionhandler.md]: Thanks for finding. Fixed.

**Review by denali49 — APPROVED** [review]
lgtm

**Review by denyeart — APPROVED** [review]

**Pull request merged**